client/actions/loginActions.js

```diff
@@ -19,9 +17,7 @@ export function logoutUser() {
             .then((response) => {
                 dispatch(logoutAction(response.data))
             })
-            .catch((err) => {
-                console.log("error =====>>>", err);
-            })
+            .catch({/** */})
```

Comment:
please don't swallow exceptions.
use this instead
catch((err) => { console.error(err);}